src/output-gazetteer.hpp

```diff
@@ -108,23 +107,19 @@ class output_gazetteer_t : public output_t
 
     /// Delete all places that are not covered by the current style results.
     void delete_unused_classes(char const *osm_type, osmid_t osm_id);
-    /// Delete all places for the given OSM object.
-    void delete_place(char const *osm_type, osmid_t osm_id);
     int process_node(osmium::Node const &node);
     int process_way(osmium::Way *way);
     int process_relation(osmium::Relation const &rel);
     void connect();
 
-    void prepare_query_conn() const;
-
     void delete_unused_full(char const *osm_type, osmid_t osm_id)
     {
```

Comment:
Why is the parameter here a `char const *`, but `delete_object()` uses a `char`?

Reply:
It was historically easier to use. No longer necessary, made it a char parameter.